draft-ietf-tcpm-converters.mkd

```diff
@@ -773,12 +767,12 @@ always ends on a 32 bits boundary after being padded with zeros.
 
 
 If a Transport Converter receives a Connect TLV with a non-empty TCP options
-field, and the Converter acceptss to process the request, it SHALL present those options to the destination
```

Comment:
Just for consistency, MUST is used everywhere else

Reply:
OK